app/services/intercom_service.rb

```diff
@@ -8,14 +8,15 @@ def self.create_message(email_address:, phone_number:, body:, client:, has_docum
     return nil if body.blank?
 
     contact = contact_from_client(client) || contact_from_email(email_address) || contact_from_sms(phone_number)
+    most_recent_convo = most_recent_conversation(contact.id)
 
-    if contact.present? && most_recent_conversation(contact.id).present?
+    if contact.present? && most_recent_convo.present?
       # Per https://developers.intercom.com/intercom-api-reference/reference/reply-to-a-conversation
       # type is always user and message_type is always comment.
       Rails.logger.info("Replying to intercom conversation -- intercom contact id ##{contact.id}")
       intercom_api(:conversations,
-                   :reply,
```

Comment:
this seems to work in most cases but for some clients were their conversations have been close for a long time it doesn't